src/signals_notebook/entities/parallel_experiment/parallel_experiment.py

```diff
@@ -132,3 +134,28 @@ def get_html(self) -> str:
         log.info('Html template for %s:%s has been rendered.', self.__class__.__name__, self.eid)
 
         return template.render(data=data)
+
+    @classmethod
+    def load(cls, path: str, fs_handler: FSHandler, notebook: Notebook) -> None:
+        from signals_notebook.item_mapper import ItemMapper
+
+        metadata = json.loads(fs_handler.read(fs_handler.join_path(path, 'metadata.json')))
+        experiment = cls.create(
+            notebook=notebook, name=metadata['name'], description=metadata['description'], force=True
+        )
+        experiment_children = [
+            child for child in experiment.get_children() if child.type != EntityType.SUB_EXPERIMENT_SUMMARY
+        ]
+
+        for child_entity in experiment_children:
+            child_entity.delete()
```

Comment:
hmm it finally works!?

Reply:
the only entity which cannot be deleted is SubExp, am I right?

Reply:
nop, subexpsummary